### PR TITLE
samples(tabs): Adding 2 tab samples

### PR DESCRIPTION
**Pull request by astaev**
Related to #2581
Adding 2 tab samples
-TabsAlignment
-TabHeader Prefix/Suffix

### CODE REVIEW

File: src/app/layouts/layouts-routes-data.ts [hunk 1]
```diff
@@ -38,6 +38,8 @@ export const layoutsRoutesData = {
     'tabs-sample-3': { displayName: 'Tabs Customizing', parentName: 'Tabs' },
     'tabs-sample-4': { displayName: 'Tabs Routing', parentName: 'Tabs' },
     'tabs-style': { displayName: 'Tabs Styling', parentName: 'Tabs' },
+    'tabs-alignment': { displayName: 'Tabs Alignment', parentName: 'Tabs' },
+    'tabs-header-prefix-suffix': { displayName: 'Tabs Suffix/Prefix', parentName: 'Tabs' },
```
Comment (gedinakova) [inline src/app/layouts/layouts-routes-data.ts:42]: Let's make it 'Tabs Prefix/Suffix'

File: src/app/layouts/tabs/tabs-header-prefix-suffix/tabs-header-prefix-suffix.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<igx-tabs>
+    <igx-tab-item>
+        <igx-tab-header>
+            <span igxPrefix>Tab</span>
```
Comment (gedinakova) [inline src/app/layouts/tabs/tabs-header-prefix-suffix/tabs-header-prefix-suffix.html:4]: I'd rather have the IG logo as a prefix and a close button as a suffix.

**Review by gedinakova — CHANGES_REQUESTED** [review]

File: src/app/layouts/layouts-routes-data.ts (hunk 1 above)
Comment (astaev) [reply src/app/layouts/layouts-routes-data.ts]: Fixed

File: src/app/layouts/tabs/tabs-header-prefix-suffix/tabs-header-prefix-suffix.html (hunk 2 above)
Comment (astaev) [reply src/app/layouts/tabs/tabs-header-prefix-suffix/tabs-header-prefix-suffix.html]: Changed

**Comment by gedinakova** [comment]
@astaev I suggest we use the smaller IG icon logo (https://static.infragistics.com/marketing/Website/General/IG-icon.svg) instead of IgniteUI log (https://static.infragistics.com/marketing/Website/products/ignite-ui-landing/ignite-ui-logo.svg), because the latter has text and doesn't align properly with the tab header label.

File: src/app/layouts/tabs/tabs-header-prefix-suffix/tabs-header-prefix-suffix.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-tabs-header-prefix-suffix',
+    templateUrl: './tabs-header-prefix-suffix.html'
+})
+export class TabsHeaderPrefixSuffixComponent {
+    public tabs = [{
+        image: 'https://static.infragistics.com/marketing/Website/products/ignite-ui-landing/ignite-ui-logo.svg',
```
Comment (gedinakova) [inline src/app/layouts/tabs/tabs-header-prefix-suffix/tabs-header-prefix-suffix.ts:9]: Replace with IG logo.

**Review by gedinakova — CHANGES_REQUESTED** [review]

**Review by gedinakova — APPROVED** [review]

**Pull request merged**